README.md

```diff
@@ -57,6 +57,7 @@ Currently, supported tools include:
   * [Samtools](http://www.htslib.org) (`stats`, `flagstats`, `idxstats`, `rmdup`)
   * [SnpEff](http://snpeff.sourceforge.net/)
   * [Subread featureCounts](http://bioinf.wehi.edu.au/featureCounts/)
+  * [Slamdunk](https://github.com/t-neumann/slamdunk)
```

Comment:
I rewrote the readme file the other day sorry, to take up less space with the list of modules. If you could pull my changes and update that would be great. Also, I try to make the list alphabetically sorted.